src/createServer.js

```diff
@@ -1,13 +1,67 @@
 'use strict';
 
-// const express = require('express');
-
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const express = require('express');
+  const fs = require('fs');
+  const path = require('path');
+
+  // Paths:
+  const publicDirPath = path.join(__dirname, 'public');
+  // Port:
+  // const PORT = process.env.PORT || 5000;
+  // App setup:
+  const app = express();
+  const usersController = require('./usersAPI/users-controller');
+  const expensesController = require('./expensesAPI/expenses-controller');
```

Comment:
will be better to create separate routes file for users and for expenses
call there needed controller, describe all possible routes (also separated, in one file - all routes for users, in the second one - all routes for expenses)

and then just call use 2 times for 2 routes in this main function